tools/ui

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$1" == "run" ]; then
```

Comment:
Optional: Use of `case` statement may be better especially when you add more actions later.